xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3604,12 +3604,16 @@ enum CudnnfMHAUid {
   MASK_ID,
   ZERO_VAL_ID,
   ONE_VAL_ID,
+  NEG_INFINITY_ID,
```

Comment:
Is it possible to factor out SE changes into a separate PR?

Reply:
Sure. Let me get one. So we keep 2 PR one for SE and one for the rest?

Reply:
Generally the more the better, as long as they are self-contained. So at least 2. Not sure if it's possible to subdivide it further.